repository/src/main/resources/db/changelog/db.changelog-rdm-1240.1.xml

```diff
@@ -0,0 +1,13 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="rdm-1240-1" author="giri.benadikar@hmcts.net">
+        <addColumn tableName="case_field">
+            <column name="data_field_type" type="varchar(10)" defaultValue="CASE_DATA">
```

Comment:
I believe it would be nicer to create an enum type first, then type this column with the enum instead of `varchar(10)`

Reply:
Done